### DISCUSSION TITLE
The httpasyncclient request was incorrectly marked in the Trace panel

### DISCUSSION THREAD

**Discussion by ywxc1997** [apache/skywalking#11206] [Q&A]
**Basic components:**
    Skywalking: 9.3.0
    Java agent: 8.16.0
    Elasticsearch java: 8.6.1
    Httpasyncclient: 4.1.3
**Problem:**
When I successfully request elastic, the request httpasyncclient/local in the trace is always marked as an error
I tried to lower the version of skywalking Java, and out of the three versions of 8.16, 8.14, and 8.9, only **8.9.0** did not encounter the above issue
![image](https://github.com/apache/skywalking/assets/55086154/bb776891-e1ba-4795-b4ab-ca238eb1f1fd)
![image](https://github.com/apache/skywalking/assets/55086154/37e5d199-5576-4c29-a60c-6e7894b1fec6)
![image](https://github.com/apache/skywalking/assets/55086154/fddea3e6-b39a-4b5d-802f-6d36c486ee91)
There is no relevant output in both the OAP and agent logs
Do I need to make some configurations in the new version？

**Comment by ywxc1997** [comment]
I tried to replace the plugin apm httpasyncclient-4.x-plugin-8.9.0.jar from 8.9.0 with version 8.16.0. The HttpAsyncClient/local in the target panel no longer reports errors, but there is no http.status_code Field
Are there any differences between these two versions of plugins?
Is this a bug？
![image](https://github.com/apache/skywalking/assets/55086154/1cc35253-a525-4c70-968a-958f7ea68e3a)

**Comment by wu-sheng** [comment]
The source codes are there, you could check and debug on your own. It is hard to tell as most of plugins were not made by maintainers.

**Reply by ywxc1997** [reply]
Thank you
But I am not a Java developer, can I turn this into an issue?

**Reply by wu-sheng** [reply]
We would not accept this. ASF is a dev volunteer drive community. 

You could leave this discussion open here, to see whether someone wants to help, but we would not track this as an issue until someone commits he/she will fix this(if this is a bug).
